google/utils_test.go

```diff
@@ -93,3 +93,11 @@ func TestIpCidrRangeDiffSuppress(t *testing.T) {
 		}
 	}
 }
+
+func TestRfc3339TimeDiffSuppress(t *testing.T) {
+	old := "2:00"
+	new := "02:00"
+	if rfc3339TimeDiffSuppress("time", old, new, nil) == false {
```

Comment:
Add also a test case to make sure it doesn't suppress a diff for the following cases:
2:00 and 3:00
02:00 and 3:00
2:00 and 03:00

I suggest you use a table driven test approach like `TestIpCidrRangeDiffSuppress` in this file.